Taskfile.yml

```diff
@@ -104,8 +104,8 @@ tasks:
 
   test-oidc-remote: bash etc/run-oidc-remote-test.sh
 
-  test-atlas-connect: 
-    - go test -v -run ^TestAtlas$ go.mongodb.org/mongo-driver/v2/internal/cmd/testatlas -args "$ATLAS_REPL" "$ATLAS_SHRD" "$ATLAS_FREE" "$ATLAS_TLS11" "$ATLAS_TLS12" "$ATLAS_SERVERLESS" "$ATLAS_SRV_REPL" "$ATLAS_SRV_SHRD" "$ATLAS_SRV_FREE" "$ATLAS_SRV_TLS11" "$ATLAS_SRV_TLS12" "$ATLAS_SRV_SERVERLESS" >> test.suite
```

Comment:
It seems none of the URI env vars were previously populated, effectively making this test a no-op. It's unclear when that change happened because the test passes silently.

Reply:
This is not quite accurate. These tests do pass and are not skipped on the correct task: https://parsley.mongodb.com/test/mongo_go_driver_atlas_test_atlas_test_patch_5b79d946e0414b30e7a4ce53beedd43abef23eb2_689a9b41febc2d0007ffba2d_25_08_12_01_39_16/0/f9ccdfa9f8beaef7f7d584890b0b7de3?bookmarks=0,31&shareLine=1

Reply:
Ah, I see that now, thanks for the clarification!